Apeswap-Bots/APESWAP-BOT-2/src/agent.ts

```diff
@@ -0,0 +1,23 @@
+import { tokenType, APESWAP_GNANA, includeAndExcludeFunctions, createFinding, functionDetails } from "./utils";
+
+import { Finding, HandleTransaction, TransactionEvent } from "forta-agent";
+
+function provideBotHandler(tokenInfo: tokenType, functionAbi: string | string[]): HandleTransaction {
```

Comment:
```suggestion
function provideBotHandler(tokenInfo: tokenType): HandleTransaction {
```
`functionAbi` is not a variable on this bot, it is a fixed value, passing a different Abi will not work as the `createFinding` function is expecting only include and exclude calls. So instead of passing the Abi as param here, use the global imported value directly in line 9